js_modules/dagster-ui/packages/ui-core/src/overview/OverviewJobsTable.tsx

```diff
@@ -71,41 +74,69 @@ export const OverviewJobsTable = ({repos}: Props) => {
   return (
     <div style={{overflow: 'hidden'}}>
       <Container ref={parentRef}>
-        <VirtualizedJobHeader />
+        {flagUseNewObserveUIs ? null : <VirtualizedJobHeader />}
         <Inner $totalHeight={totalHeight}>
-          {items.map(({index, key, size, start}) => {
-            const row: RowType = flattened[index]!;
-            const type = row!.type;
-            return type === 'header' ? (
-              <RepoRow
-                repoAddress={row.repoAddress}
-                key={key}
-                height={size}
-                start={start}
-                onToggle={onToggle}
-                onToggleAll={onToggleAll}
-                expanded={expandedKeys.includes(repoAddressAsHumanString(row.repoAddress))}
-                showLocation={duplicateRepoNames.has(row.repoAddress.name)}
-                rightElement={
-                  <Tooltip
-                    content={row.jobCount === 1 ? '1 job' : `${row.jobCount} jobs`}
-                    placement="top"
-                  >
-                    <Tag>{row.jobCount}</Tag>
-                  </Tooltip>
-                }
-              />
-            ) : (
-              <VirtualizedJobRow
-                key={key}
-                name={row.name}
-                isJob={row.isJob}
-                repoAddress={row.repoAddress}
-                height={size}
-                start={start}
-              />
-            );
-          })}
+          <div
+            style={{
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              width: '100%',
+              transform: `translateY(${items[0]?.start ?? 0}px)`,
+            }}
```

Comment:
This windowing approach matches what is done in tanstack example code.

Reply:
Is that a new change? Seems kind of weird but I guess it means you only need to dynamically modify the CSS of a single element.

Reply:
Might be new, I'm not sure. It looked a little cleaner so I gave it a try and it seemed pretty good.